plugins/input/postgis/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.15.0)
+project(input-postgis)
+
+find_package(PostgreSQL REQUIRED)
+
```

Comment:
```diff
diff --git a/plugins/input/postgis/CMakeLists.txt b/plugins/input/postgis/CMakeLists.txt
index ba3192a98..0b804b761 100644
--- a/plugins/input/postgis/CMakeLists.txt
+++ b/plugins/input/postgis/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.15.0)
+cmake_minimum_required(VERSION 3.19)
 project(input-postgis)
 
 find_package(PostgreSQL REQUIRED)
@@ -9,6 +9,7 @@ add_library(input-postgis SHARED
 )
 add_library(mapnik::plugin::input::postgis ALIAS input-postgis)
 
+target_include_directories(input-postgis PRIVATE ${PostgreSQL_INCLUDE_DIR})
 target_link_libraries(input-postgis PRIVATE
     mapnik::headers
     mapnik::mapnik
```

postgis.input needs postgresql/postgis headers

Reply:
ah missed that one.

Reply:
changed with 1f73d1c

Reply:
Hmm.. it works for me without `target_include_directories(input-postgis PRIVATE ${PostgreSQL_INCLUDE_DIR})`. It must have been something with my settings.